### ISSUE TITLE
Be able to browser liked and watched projects

### ISSUE THREAD

**Issue by willeastcott** [playcanvas/editor#907]
The super-cool new project dashboard allows you to browse and filter projects to a basic extent:

![image](https://user-images.githubusercontent.com/697563/197874468-1ef17010-9978-4b21-9ca1-813760a09600.png)

It would be great if you could also view liked and watched projects as well from here.

**Comment by rgsaura** [comment]
Yes! Please make this available for us developers. Playcanvas is based on a lot of collaboration & forking so we need to be able to organize ourselves inside the system.
